app/scrambler.py

```diff
@@ -1,17 +1,69 @@
+from random import choice, shuffle
+
 from picture import Picture
+from tile import Tile
 
 
 class Scrambler:
     """Scramble a picture in different ways."""
 
-    def scramble_rows(self, picture: Picture) -> None:
+    @classmethod
+    def scramble_rows(cls, picture: Picture) -> None:
         """Split an Image up into rows an rearranges them."""
-        raise NotImplementedError
+        num_tiles = 8
+        image_size = picture.image.size
+        tile_size = (image_size[0], int(image_size[1] / num_tiles))
 
-    def scramble_grid(self, picture: Picture) -> None:
-        """Split an Image up into tiles an rearranges them."""
-        raise NotImplementedError
+        for y in range(num_tiles):
+            x0, y0, x1, y1 = (0, tile_size[1] * y, tile_size[0], tile_size[1] * (y + 1))
```

Comment:
I'm not too fond of this compact assignment, could this be done line per variable?

Reply:
I made some changes so its more readable now